stablehlo/tests/legalize_stablehlo_to_vhlo.mlir

```diff
@@ -1227,6 +1227,13 @@ func.func @op_replica_id() -> tensor<ui32> {
 }
 // CHECK-LABEL: "op_replica_id"
 
+func.func @op_partition_id() -> tensor<ui32> {
+  // CHECK: "vhlo.partition_id"() : () -> tensor<ui32>
+  %0 = "stablehlo.partition_id"() : () -> tensor<ui32>
+  func.return %0 : tensor<ui32>
+}
```

Comment:
Could you also add a TP to `vhlo_to_version_downgrade_invalid.mlir`. Can use this same snippet, just check for error downgrading since it didn't exist in the original opset.